### PR TITLE
Refs #17; Added the necessary information in package.json

### PR DESCRIPTION
**Pull request by rsjavier14**
Warnings removed after running the command "npm install"

### CODE REVIEW

**Comment by edipox** [comment]
Hi @rsjavier14 Thank you for your PR!

File: package.json [hunk 1]
```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
+  "main": "webpack.config.js",  
+  "version": "",
+  "directories": {
+    "test": "test"
+  },
+  "keywords": [],
+  "preferGlobal": true,
```
Comment (edipox) [inline package.json]: @rsjavier14 What is this option for? Why have you added it?

File: package.json [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+  "contributors": [
+    {
+      "name": "Javier Rios Selent",
```
Comment (edipox) [inline package.json]: @rsjavier14 Is this "contributors" field required? 
Your PR will probably fix the issue #17 but I think we have to find a way to do one of the following: put here all the contributors or put something like _iporaitech_
What do you think @hisa?

File: package.json [hunk 3]
```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
```
Comment (hisapy) [inline package.json]: Please add _"**Javascript dependencies for pwr2-docker, a** Docker image based on ...."_ in the description.

File: package.json [hunk 4]
```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
```
Comment (hisapy) [inline package.json]: Do you guys think that pwr2 is enough? ... what about pwr2-cli or pwr2-ui ? ... Just imagine all of the frontend stuff can be packaged with NPM...

File: package.json [hunk 5]
```diff
@@ -1,4 +1,25 @@
...
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
+  "main": "webpack.config.js",  
+  "version": "",
+  "directories": {
```
Comment (hisapy) [inline package.json]: What is this for ? ...

File: package.json [hunk 6]
```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
+  "description": "Docker image based on Ubuntu providing a base setup for a Phoenix+Webpack+Relay+React project, with some sugar and conventions to develop and build your own web applications.",
+  "main": "webpack.config.js",  
+  "version": "",
```
Comment (hisapy) [inline package.json]: version should be the same as the last git tag (or release).

File: package.json (hunk 1 above)
Comment (hisapy) [reply package.json]: Same question as @edipox. As far as I know, we don't prefer global.

File: package.json (hunk 2 above)
Comment (hisapy) [reply package.json]: I think 

``` json
{
  "name": "Iporaitech and others",
  "url": "https://github.com/iporaitech/pwr2-docker/graphs/contributors"
}
```

With no specific email address.

File: package.json [hunk 7]
```diff
@@ -28,5 +49,6 @@
     "react-dom": "^15.3.0",
     "react-relay": "^0.9.2",
     "sync-request": "^3.0.1"
-  }
+  },
+  "license" : "MIT"
```
Comment (hisapy) [inline package.json:47]: I think this should be a link to our license file https://github.com/iporaitech/pwr2-docker/blob/master/LICENSE ... Please correct me if I'm wrong.

Comment (rsjavier14) [reply package.json:47]: you're right..

File: package.json (hunk 6 above)
Comment (rsjavier14) [reply package.json]: sorry, i fix it.

File: package.json (hunk 3 above)
Comment (rsjavier14) [reply package.json]: ok, i added.

File: package.json (hunk 2 above)
Comment (rsjavier14) [reply package.json]: great!!

File: package.json (hunk 4 above)
Comment (rsjavier14) [reply package.json]: any suggestions?

File: package.json (hunk 1 above)
Comment (rsjavier14) [reply package.json]: sorry , I have little experience with these settings. Looking at the documentation thought I could use. I fix it?

File: package.json (hunk 7 above)
Comment (rsjavier14) [reply package.json:47]: I correct myself , you're wrong. That is not a valid SPDX license expression [http://spdx.org/](http://spdx.org/)
Please look at this site, where you will find licenses SPDX [https://spdx.org/licenses/index.html](https://spdx.org/licenses/index.html).

**Pull request merged**